### ISSUE TITLE
JSX-A11y Plugin Compatibility with WCAG 2.2

### ISSUE THREAD

**Issue by jashan777** [jsx-eslint/eslint-plugin-jsx-a11y#1008]
Hello! I’m opening this issue to inquire about the compatibility of the current plugin rules with WCAG 2.2. Are there any plans to update the plugin to align with the WCAG 2.2 guidelines?

**Comment by jashan777** [comment]
thanks for such a speedy reply, I was looking at the rules documentation and the rules follow Wcag 2.1 Accessibility guidelines  and there are some points I noticed (related to Wcag, in context to rules documentation) 
1. The jsx-a11y/scope rule: 4.1.1 Accessibility guidelines Parsing is obsolete and removed from WCAG 2.2.
2. No other rule change.
with the above points, is there a possibility that the plugin rules documentation can be updated to point to WCAG 2.2?

**Comment by ljharb** [comment]
That doesn't mean the scope rule isn't useful - that WCAG no longer requires violating it to be a parsing error doesn't mean it's *correct* to violate it.

**Comment by jashan777** [comment]
I’m completely with you on this—I was just bringing up the point with respect to WCAG 2.2 standards and not to question the value of the rule itself.

**Comment by ljharb** [comment]
So, it sounds like there's nothing we need to do to be compliant with WCAG 2.2.
